packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.css

```diff
@@ -1 +1 @@
-@import 'c/searchBoxStyle';
+@import '../searchBoxStyle/searchBoxStyle.css';
```

Comment:
Is it still importing correctly?

Reply:
yeah nope, will have to revert and figure out something else